catalyst/contrib/dl/callbacks/optuna_callback.py

```diff
@@ -0,0 +1,68 @@
+import optuna
+
+from catalyst.core import Callback, CallbackOrder, IRunner
+
+
+class OptunaCallback(Callback):
```

Comment:
let's add it to the docs
https://github.com/catalyst-team/catalyst/blob/master/docs/api/contrib.rst